### PR TITLE
Feature profile users

### PR DESCRIPTION
**Pull request by OmKakatkar**
# Feature

## Additions
* Add few test users
* Add profile route with profile edit functionality
* User can be followed and unfollowed

### CODE REVIEW

**Review by Logan1x — APPROVED** [review]

File: src/frontend/constants/api-constant.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
 export const API_LOGIN = "/api/auth/login";
 export const API_SIGNUP = "/api/auth/signup";
 export const API_USER_EDIT = "/api/users/edit";
+export const API_USERS = "/api/users/";
+export const API_FOLLOW = "/api/users/follow";
+export const API_UNFOLLOW = "/api/users/unfollow";
```
Comment (parulgupta2001) [inline src/frontend/constants/api-constant.js:6]: export all these from one file ie. index.js

**Review by parulgupta2001 — APPROVED** [review]
code is well structured

File: src/frontend/constants/api-constant.js (hunk 1 above)
Comment (OmKakatkar) [reply src/frontend/constants/api-constant.js:6]: Thanks for the review. These are API constants and hence exported form a single file.

**Comment by SadiqZak** [comment]
The code is clean and is good to merge

**Pull request merged**